.gitignore

```diff
@@ -1,9 +1,40 @@
+# Environments
```

Comment:
Do we have to clutter up the gitignore? Do you really have all of these things that need to be ignored? Or is this from some default list? I'd rather only ignore what's strictly necessary.